modules/nf-core/shapeit5/phaserare/main.nf

```diff
@@ -0,0 +1,62 @@
+process SHAPEIT5_PHASERARE {
+    tag "$meta.id"
+    label 'process_low'
+
+    beforeScript  """
+    if cat /proc/cpuinfo | grep avx2 -q
+    then
+        echo "Feature AVX2 present"
+    else
+        echo "Feature AVX2 not present on node"
+        exit 1
+    fi
+    """
+
+    conda "bioconda::shapeit5=1.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/shapeit5:1.0.0--h0c8ee15_0':
+        'quay.io/biocontainers/shapeit5:1.0.0--h0c8ee15_0' }"
+
+    input:
+        tuple val(meta) , path(input_plain), path(input_plain_index), val(input_region), path(pedigree)
+        tuple val(meta2), path(scaffold)   , path(scaffold_index)   , val(scaffold_region)
+        tuple val(meta3), path(map)
+
+    output:
+        tuple val(meta), path("*.{vcf,bcf,vcf.gz,bcf.gz}"), emit: phased_variant
+        path "versions.yml"                               , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
+    def prefix = task.ext.prefix ?: "${meta.id}_${input_region.replace(":","_")}"
+    def suffix = task.ext.suffix ?: "vcf.gz"
+
+    def map_command       = map       ? "--map $map"             : ""
+    def pedigree_command  = pedigree  ? "--pedigree $pedigree"   : ""
+
+    meta.put("SHAPEIT5_PHASERARE", ["map":"", "scaffold":""])
+    meta.SHAPEIT5_PHASERARE.map       = map       ? meta2 :"None"
+    meta.SHAPEIT5_PHASERARE.scaffold  = scaffold  ? meta3 :"None"
```

Comment:
This is something that should be done on a pipeline level. It's preffered to not change any meta maps in modules